packages/knack-avro/readme.md

```diff
@@ -10,11 +10,71 @@
   <a href="https://github.com/xojs/xo"><img src="https://img.shields.io/badge/code_style-XO-5ed9c7.svg"></a>
 </p>
 
-## Usage
-
 <b>first things first...</b>
 
 ```shell
 $ npm i @optum/knack-avro
 ```
 
+<b>methods</b>
+
+- toAvroBuffer: (data, schema, schemaId)
+  + get avro encoded buffer from json
+    * data: `Object`  json object to encode 
+    * schema: `Object or String`  schema to use when encoding the data
+    * schemaId: `Number` schemaId of the schema from the schema registry
+- fromAvroBufferd: (schema, buffer)
```

Comment:
`fromAvroBufferd`, missing an 'e' or does it have an extra 'd'?

Reply:
fixed in latest commit